src/test/java/com/coherent/task40/TestConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.coherent.task40;
+
+public class TestConstants {
+    public static String YANDEX_LOGIN_PAGE = "https://mail.yandex.com";
+    public static String DOWNLOAD_USER_PAGE = "https://demo.seleniumeasy.com/bootstrap-download-progress-demo.html";
+    public static String SORT_TABLE_PAGE = "https://demo.seleniumeasy.com/table-sort-search-demo.html";
+    public static String ALERTS_PAGE = "https://demo.seleniumeasy.com/javascript-alert-box-demo.html";
+    public static String DROPDOWN_PAGE = "https://demo.seleniumeasy.com/basic-select-dropdown-demo.html";
+    public static String DYNAMIC_DATA_PAGE = "https://demo.seleniumeasy.com/dynamic-data-loading-demo.html";
```

Comment:
You also should add **final** as they are constants.

Reply:
done